avro2py/codegen.py

```diff
@@ -363,10 +393,15 @@ def bridge_namespaces(from_namespace: List[str], to_namespace: List[str], target
         remaining_to_components = [c for c in paired_remaining_to_components if c is not None]
 
         if remaining_from_components:
+            from_namespace_as_str = '.'.join(from_namespace)
+            if from_namespace_as_str in namespaces_with_children:
+                level = len(remaining_from_components) + 1
```

Comment:
Can you add a comment here explaining why we do this?

Reply:
Yes I can, although I think the situation is pretty specific so I'm not sure how clear the comment will be.